### DISCUSSION TITLE
为什么xmake支持添加任意版本的依赖，但xrepo不支持

### DISCUSSION THREAD

**Discussion by iGwkang** [xmake-io/xmake#6018] [Q&A]
`xmake`可以通过 `add_requires `指定`{verify = false} `添加任意版本的包，
但是xrepo没有这样的参数可以指定。xrepo只能指定configs

**Comment by waruqi** [comment]
verify 原本就不推荐用，所以暂时不支持

**Reply by iGwkang** [reply]
好的，没事，我可以让xrepo指定xmake.lua，然后在xmake.lua里面加verify
